bk-config/README.md

```diff
@@ -0,0 +1,43 @@
+
+This container 
+
+
+```bash
+
+docker build -t sagecontinuum/bk-config .
+
+```
+
+
+
+# Create CA and beekeeper key-pairs and certificates and registration key (NOT registration cert)
+
+```bash
+docker run -ti --rm --network beekeeper_default --name bk-config -v beekeeper-config_bk-secrets:/usr/lib/sage/ sagecontinuum/bk-config
+
+
+init-keys.sh
+```
+
+Afterwards you can start beekeeper.
+
+
+# create client files (e.g. for a node) 
+
```

Comment:
I would remove the sentence "Afterwards you can start beekeeper." and add a sentence here stating that you must have `beekeper` running to create the client files.

and to be honest I was expecting the `create_client_files.sh` script to be run from within the bk-config container.  I have added more detailed comments to the `create_client_files.sh` script.

Reply:
beekeeper should not have to be running to create the initial files, unless there is a bug I missed.